l10n/sv-SE/viewer.properties

```diff
@@ -137,17 +137,20 @@ print_progress_close=Avbryt
 # (the _label strings are alt text for the buttons, the .title strings are
 # tooltips)
 toggle_sidebar.title=Visa/dölj sidofält
-toggle_sidebar_notification.title=Visa/dölj sidofält (dokument innehåller översikt/bilagor)
+toggle_sidebar_notification2.title=Visa/dölj sidofält (dokument innehåller översikt/bilagor/lager)
 toggle_sidebar_label=Visa/dölj sidofält
 document_outline.title=Visa dokumentdisposition (dubbelklicka för att expandera/komprimera alla objekt)
 document_outline_label=Dokumentöversikt
 attachments.title=Visa Bilagor
 attachments_label=Bilagor
+layers.title=Visa lager (dubbelklicka för att återställa alla lager till ursrungligt läge)
+layers_label=Lager
 thumbs.title=Visa miniatyrer
 thumbs_label=Miniatyrer
 findbar.title=Sök i dokument
 findbar_label=Sök
 
+additional_layers=Ytterliggare lager
```

Comment:
There are some tiny typos in this Swedish file. Or will these be overwritten when strings arrive from the translation process?

Reply:
Yes, there's obviously an embarrassing typo on this line; fixed now :-)
(It seems that a bad spell-checker might have "helped" me here.)

Generally speaking, we're importing all translations from Firefox and all `l10n/` files are thus updated/overwritten regularly; any pre-existing bugs thus need to be filed against the Firefox localization in Bugzilla.
(When adding *new* strings, I simply find it helpful to add not just the default language for testing purposes.)